src/items/menu.js

```diff
@@ -109,6 +109,27 @@ const makeTree = (core, __, metadata) => {
  */
 export default class MenuPanelItem extends PanelItem {
 
+  init(state, actions) {
+    const onkeydown = ev => {
+      const checkKeys = this.core.config('panel.menu.keys', 'Alt+a').toLowerCase().split('+');
+      const modifierNames =  ['ctrl', 'shift', 'alt', 'meta'];
+      const keyName = String(ev.key).toLowerCase();
+      const validKeypress = checkKeys.every(k => modifierNames.indexOf(k) !== -1 ? ev[k + 'Key'] : keyName === k);
```

Comment:
When using the ternary, I'd recommend styling it like this. Much easier to read :)

```
const validKeypress = checkKeys.every(k => modifierNames.indexOf(k) !== -1
  ? ev[k + 'Key']
  : keyName === k);
```